lxc/copy.go

```diff
@@ -77,6 +79,11 @@ func (c *cmdCopy) copyContainer(conf *config.Config, sourceResource string,
 		return fmt.Errorf(i18n.G("You must specify a destination container name when using --target"))
 	}
 
+	// Check that a destination container was specified, if --refresh is passed.
+	if destName == "" && c.flagRefresh {
+		return fmt.Errorf(i18n.G("You must specify a destination container name when using --refresh"))
```

Comment:
Hmm, why not assume it's the same as the source?

`lxc copy blah remote: --refresh` should be fine right?